### DISCUSSION TITLE
Sensors for DDR5, GPU Memory Junction, and GPU Hotspot

### DISCUSSION THREAD

**Discussion by louispool** [Rem0o/FanControl.Releases#552] [Q&A]
![image](https://user-images.githubusercontent.com/19853923/159677234-59e5eba4-56fa-4463-8bd7-572be11b43a3.png)

Hi guys,

Great software, thanks so much,  and I want you to know that I already donated once and will continue to do so as more features become available. 

I  really appreciate the capability to define sensor "groups" where  I can include the GPU, CPU and  VRM  temperatures in a single group that will drive my fans - this is particularly important in water cooled setups, like my own, where all the components are in a single loop, and you can't rely on just the CPU temperature to signal when to increase the pump and/or radiator fan speeds.

That being said, I do miss a few crucial sensors: 

- the temps on the memory (reported by HWInfo as the SPD Hub Temperature - see attached image)
- the GPU memory junction temperature 
- and the GPU Hot Spot Temperature

Does anyone know whether there are plugins available that would expose these sensors to FanControl?

For reference, I have an MSI Z690 Carbon Mobo with an Intel 12900K CPU and an Nvidia 3090 GFX Card.

**Comment by boroborob** [comment]
Hey!

Open the sensor settings from the menu and disable the NvApiWrapper one and use the other one provided by LibreHardwareMonitor instead.
I can't speak for the SPD Hub, but if it's recognized by LibreHardwareMonitor it may work if Memory is enabled as well.

![Képernyőfelvétel (8)](https://user-images.githubusercontent.com/32228126/161402813-29788972-1538-44c9-bd14-2a972e01440e.png)

**Reply by boroborob** [reply]
also there is a HWInfo plugin, but you need the PRO version for it or it only works for 12-hours.

    "Plugins

    Plugin system, see [Plugins wiki](https://github.com/Rem0o/FanControl.Releases/wiki/Plugins) and
    https://github.com/Rem0o/FanControl.HWInfo
    https://github.com/Rem0o/FanControl.DellPlugin
    https://github.com/medevil84/FanControl.AquacomputerHighFlowNext"

**Reply by louispool** [reply]
Hi boroborob,

Thanks for replying! Unfortunately, disabling NvApiWrapper doesn't do anything except remove the GPU sensor. 

But I'll try out the HWInfo Plugin - thanks for the heads up - somehow I didn't see the example listed on the Plugins wiki. I guess I was looking for some kind of an index of available plugins.

**Reply by louispool** [reply]
Yeah, unfortunately that doesn't work either. The plugin picks up a few Speed Sensors exposed by HWInfo, but none of the temperature sensors that I'm interested in, as far as I can see.

**Reply by Rem0o** [reply]
> NvApiWrapper doesn't do anything except remove the GPU sensor.

Make sure the other GPU checkbox is ticked, it should pick up some GPU sensors, those won't be the "same" as the nviapiwrapper ones, and will break your current config. But some should appear.
